### DISCUSSION TITLE
Multi-Select Dropdown

### DISCUSSION THREAD

**Discussion by jasonhill** [twbs/bootstrap#32110] [Q&A]
Are there any plans to support multi-select dropdowns natively in BS5? So far, we have only found plugins that require jQuery and we want to stay native.

**Comment by FabianKoestring** [comment]
@jasonhill Did you found an vanila js alternativ?

**Reply by jasonhill** [reply]
I found this version which was really lightweight but haven't tested it thoroughly yet: https://github.com/kleimaj/multiSelect.js.
